test/util/util.go

```diff
@@ -77,6 +77,40 @@ import (
 	"sigs.k8s.io/kueue/pkg/workloadslicing"
 )
 
+var DefaultTestPodSetUpdates1 = []kueue.PodSetUpdate{
```

Comment:
Ah. these are dependent on the workload, ac and prov req names, so I don't think this belongs to util.go.

Let's just define the consts in the provisioning_test.go, unexported. Think of the smallest possible scope of declaration.